sbnanalysis/core/Loader.cxx

```diff
@@ -83,7 +83,7 @@ fhicl::ParameterSet* LoadConfig(char* configfile) {
   if (configfile) {
     std::cout << "Loading configuration: " << configfile << "... ";
     config = new fhicl::ParameterSet;
-    cet::filepath_maker maker;
+    cet::filepath_lookup_after1 maker("FHICL_FILE_PATH");
```

Comment:
Would `cet::filepath_lookup_nonabsolute` be better here? I think that one will allow absolute paths but try to look up any non-absolute path.

Reply:
Yeah, that does look better -- fixed.